sync-diff-inspector/sync-diff-inspector-overview.md

```diff
@@ -106,7 +127,7 @@ skip-non-existing-table = false
 
     # (optional) Use the snapshot feature. If enabled, historical data is used for comparison.
     # snapshot = "386902609362944000"
-    # When "snapshot" is set to "auto", the last syncpoints generated by TiCDC in the upstream and downstream are used for comparison. For details, see <https://github.com/pingcap/tidb-tools/issues/663>.
+    # When "snapshot" is set to "auto", the last syncpoints generated by TiCDC in the upstream and downstream are used for comparison. For details, see <https://github.com/pingcap/tiflow/issues/663>.
```

Comment:
Is this update necessary? They look like different issues.
- https://github.com/pingcap/tidb-tools/issues/663
- https://github.com/pingcap/tiflow/issues/663